setup.py

```diff
@@ -67,9 +67,11 @@
     "requests",
     "scikit-learn",
     "scipy",
-    "scispacy",
+    # >= 0.4.0 to be compatible with spaCy 3 and let use the en_core_sci_scibert model.
+    "scispacy>=0.4.0",
```

Comment:
I tried to read through your instructions but I still do not understand the following point.

You said that this PR is not about adding new NER models yet here you are assuming we want to use `en_core_sci_scibert`.

Reply:
#294, created by @FrancescoCasalegno, assumes we want to use, or at least try, `en_core_sci_scibert`.
So, I don't assume this myself.

Note: The PR is not adding `en_core_sci_scibert`.

Reply:
Because of a).

This is said in the [release notes](https://github.com/allenai/scispacy/releases/tag/v0.4.0) of `scispaCy v0.4.0`:
> This release of scispacy is compatible with Spacy 3.

Reply:
Fine with me, thanks!

Reply:
I have removed the mention of `en_core_sci_scibert` to avoid confusions in 7d2dc96.